client/src/templates/style.scss

```diff
@@ -0,0 +1,5 @@
+.close-button {
+ color: rgb(80, 82, 86);
+ background: rgb(247, 248, 249); 
```

Comment:
Not a fan of having this class here. What do you think about moving this in `components/Button/style.scss`, or change to `secondary` button

Reply:
I agree yeah. I just moved it into the RoomTemplate